docs/about/comms_channels.qmd

```diff
@@ -2,4 +2,14 @@
 title: "Communication Channels"
 ---
 
-Coming soon!
+To stay in touch with our community you can participate in one of our 
+communication channels or join our [community meetings](calendar.html).
```

Comment:
Should we add a note here that the community meeting time and date is still TBD?

Reply:
Addressed by #29